### PR TITLE
Update assign-change-or-remove-a-phone-number-for-a-user.md

### PR DESCRIPTION
**Pull request by gosepulv**
together with the modification already requested for https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps, it can become more clear to admins (less experienced would be the target here) how to manage phone numbers from user object, since this documentation is also seen by admins not so technically experienced and while experiencing issues managing phone numbers from Teams Admin Center, customers find themselves stuck and firstline support using same documentation, may be more easily conducted to providing admins with suitable PowerShell solution following this additional PowerShell option. I also find it important to make a difference between Direct Routing and Calling Plans since both scenarios use assign, change and remove phone numbers from users

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ ms.custom:
 description: "Learn how to assign, change, or remove a work phone number for your Teams users so outside businesses and clients can call in."
 ---
 
-# Assign, change, or remove a phone number for a user
+# Assign, change, or remove a phone number for a user (Calling Plans)
 
-When you set up Calling Plans, you assign phone numbers to your users. In Microsoft Teams, the phone number you assign is listed when a user clicks **Calls**.
+When you set up Calling Plans, you assign phone numbers to your users. In Microsoft Teams, the phone number you assign is listed when a user clicks **Calls**. If you're looking for instructions to assign, change or remove a phone number from a user for a Direct Routing scenario, please consult [Enable users for Direct Routing, voice, and voicemail](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-enable-users).
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:27]:
```suggestion
When you set up Calling Plans, you assign phone numbers to your users. In Microsoft Teams, the phone number you assign is listed when a user clicks **Calls**. If you're looking for instructions to assign, change or remove a phone number from a user for a Direct Routing scenario, please see [Enable users for Direct Routing, voice, and voicemail](https://docs.microsoft.com/microsoftteams/direct-routing-enable-users).
```

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ To learn how to get Calling Plans in Teams and how much they cost, see [Teams ad
 4. To assign or change the associated emergency location, under **Emergency location**, search for and then select the location.
 5. Depending on whether you want to send an email to the user with their phone number information, turn off or turn on **Email user with telephone number information**. By default, this is on. 
 6. Click **Save**.
+
+PowerShell example provided at [Set-CsOnlineVoiceUser](https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:49]:
```suggestion
For a PowerShell example, please see [Set-CsOnlineVoiceUser](https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps).

```

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 3]
```diff
@@ -63,6 +65,8 @@ To learn how to get Calling Plans in Teams and how much they cost, see [Teams ad
 8. To assign or change the associated emergency location, under **Emergency location**, search for and then select the location.
 9. Click **Save**.
 
+PowerShell example provided at [Set-CsOnlineVoiceUser](https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:68]:
```suggestion
For a PowerShell example, please see [Set-CsOnlineVoiceUser](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps).
```

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 4]
```diff
@@ -72,7 +76,9 @@ To learn how to get Calling Plans in Teams and how much they cost, see [Teams ad
 3. On the **Phone numbers** page, select the number that you identified in step 2, and then click **Edit**.  
 4. In the **Edit** pane, under **Assigned to**, click the **X** to remove the user.
 5. Click **Save**.
-    
+
+PowerShell example provided at [Set-CsOnlineVoiceUser](https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:80]:
```suggestion
For a PowerShell example, please see [Set-CsOnlineVoiceUser](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 5]
```diff
@@ -82,3 +88,7 @@ To learn how to get Calling Plans in Teams and how much they cost, see [Teams ad
 [Emergency calling terms and conditions](/microsoftteams/emergency-calling-terms-and-conditions)
 
 [Emergency Calling disclaimer label](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/downloads/emergency-calling/emergency-calling-label-(en-us)-(v.1.0).zip?raw=true)
+
+[Set-CsOnlineVoiceUser](https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:92]:
```suggestion
[Set-CsOnlineVoiceUser](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps)
```

File: Teams/assign-change-or-remove-a-phone-number-for-a-user.md [hunk 6]
```diff
@@ -82,3 +88,7 @@ To learn how to get Calling Plans in Teams and how much they cost, see [Teams ad
...
+
+[Calling Plans for Microsoft 365](https://docs.microsoft.com/en-us/MicrosoftTeams/calling-plans-for-office-365)
```
Comment (JohanFreelancer9) [inline Teams/assign-change-or-remove-a-phone-number-for-a-user.md:94]:
```suggestion
[Calling Plans for Microsoft 365](https://docs.microsoft.com/MicrosoftTeams/calling-plans-for-office-365)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@gosepulv - could you please commit my suggestions? Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by gosepulv** [comment]
thank you @JohanFreelancer9 ;)

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**